sourceinfo/source_code_info.go

```diff
@@ -542,7 +576,7 @@ type comments interface {
 	Index(int) ast.Comment
 }
 
-var emptyComments = subComments{offs: 0, n: 0, c: ast.Comments{}}
+var emptyComments = ast.Comments{}
```

Comment:
Any benefit to exporting this from the ast package? There are a few places there that allocate an empty comments object like this.